bootstrap/postgres/init.go

```diff
@@ -83,6 +83,15 @@ func Migration() *migrate.MemoryMigrationSource {
 					`ALTER TABLE IF EXISTS channels RENAME COLUMN mainflux_channel TO magistrala_channel`,
 				},
 			},
+			{
+				Id: "configs_5",
+				Up: []string{
+					`ALTER TABLE IF EXISTS configs RENAME COLUMN owner TO domain_id`,
+					`ALTER TABLE IF EXISTS channels RENAME COLUMN owner TO domain_id`,
+					`ALTER TABLE IF EXISTS connections RENAME COLUMN channel_owner TO channel_domain_id`,
+					`ALTER TABLE IF EXISTS connections RENAME COLUMN config_owner TO config_domain_id`,
+				},
+			},
```

Comment:
I should be able top create a bootstrap config with the same name on different domains

Reply:
You should be able to, with the current codebase just as it is